src/DateTimeValidator.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace ipl\Validator;
+
+use DateTime;
+use ipl\I18n\Translation;
+
+/**
+ * Validator for date-and-time input controls
+ *
+ * @see \Icinga\Web\Form\Element\DateTimePicker For the date-and-time input control.
+ */
+class DateTimeValidator extends BaseValidator
+{
+    use Translation;
+
+    protected $local;
+
+    /**
+     * Create a new date-and-time input control validator
+     *
+     * @param bool $local
+     */
+    public function __construct($local)
+    {
+        $this->local = (bool) $local;
+    }
+
+    /**
+     * Is the date and time valid?
```

Comment:
No question mark or anything like that in descriptions please. It's a description after all, not a question or statement.